### PR TITLE
Finished map004

### PR DESCRIPTION
**Pull request by Zaeno-Eileena**
[Map004.csv](https://github.com/OfficialLayton/Captured-Als-To-Romeria-Translation/files/10696192/Map004.csv)

### CODE REVIEW

**Comment by OfficialLayton** [comment]
Thank you for your update Zaeno!

I'm terribly sorry for the late reply. I think the pull request has merge conflicts so I can't merge your changes just yet, but you should be able to resolve them by merging the main branch with your current one.

If you're having trouble send me a DM or ping and I can help ^^

File: Translation Script/Enemies.csv [hunk 1]
```diff
@@ -1,7 +1,7 @@
 "テスト用雑魚","Minions for Testing"
 "ルナ","Luna"
-"2戦目","2Battle Eye"
-"\n[1]",
+"2戦目","Battle Eye"
+"\n[1]"," can you see anything we can use?"""
```
Comment (OfficialLayton) [inline Translation Script/Enemies.csv:4]: This line doesn't look quite right. Was this intentional?

**Review by OfficialLayton — CHANGES_REQUESTED** [review]
Looking good! Just one odd line that I can see. Everything else looks great ^^

File: Translation Script/Enemies.csv (hunk 1 above)
Comment (Zaeno-Eileena) [reply Translation Script/Enemies.csv]: Yea I think I made a mistake randomly, my bad

**Comment by OfficialLayton** [comment]
All merged! Terribly sorry about the delay. Much appreciated Zaeno!

**Pull request merged**